scripts/common.sh

```diff
@@ -548,6 +548,46 @@ configure_local_api_keys() {
     print_success "API keys configured successfully"
 }
 
+generate_litellm_master_key() {
+    # If CRS_KEY_ID is already set to the default value, skip key generation
```

Comment:
This comment looks a bit out of place. I assume we are talking about `LITELLM_MASTER_KEY` here and that we proceed with key-generation if it is empty or set to default key.